applications/serial_lte_modem/overlay-ppp-without-cmux.overlay

```diff
@@ -16,5 +16,5 @@
 &uart1 {
 	status = "okay";
 	hw-flow-control;
-	current-speed = <1000000>;
+	current-speed = <115200>;
```

Comment:
I don't believe that the default PPP baudrate is mentioned in the documentation.

I had severe difficulties using pppd with 1000000 baudrate. The first attempt was successful and subsequent attempts failed. I could revisit the problem and add a comment in here.

Reply:
When it has been failing have you tried restarting SLM's PPP? I recall seeing issues similar to yours, and the UART staying disabled because the modem modules don't have any auto-recover mechanism (Could it just be added to SLM's PPP by listening to PIPE_CLOSE events and re-opening directly?), but I think that I was pretty much always able to get it working at 1M by fiddling a bit.
Anyways, I think it would probably be good to mention something if there is a persistent issue with something.

Reply:
With 1000000 baudrate, it seems that when the pppd connects for the second (or subsequent) attempt the PPP frames are received fine in the DK, but all the TX operations timeout. Setting a huge timeout (10 seconds) seemed to help with this, but this smells like a bug as you should be able to eventually transmit something in < 1 sec.

I will investigate this more next week.

Reply:
Restored the 1000000 baudrate as I could not reliably reproduce the issue with it. It is likely that the issue is somehow related to pppd or the machine running pppd and a reboot solved the issue.

I am afraid that this is something that we will be returning to at some point in future.

Reply:
Good. Well, let's keep an eye on this indeed. Again as we discussed if the UART backend were automatically restored when it goes to RX disabled it could potentially help with this.